src/style.css

```
@@ -1 +1,18 @@
 /* styles go here */
+.block {
+  margin-top: 0;
+  margin-bottom: 20px;
+}
+
+.input {
+  display: block;
+  margin: 10px 0;
+}
+
+.block > .input:first-of-type {
+  margin-top: 0;
+}
+
+.block > .input:last-of-type {
+  margin-bottom: 0;
+}
```

Don't increase specificity
```suggestion
.input:last-of-type {
  margin-bottom: 0;
}
```